authentication/certificates/service-serving-certificate.adoc

```diff
@@ -0,0 +1,24 @@
+[id="add-service-serving"]
+= Securing service traffic using service serving certificate secrets
+include::modules/common-attributes.adoc[]
+:context: service-serving-certificate
+
+toc::[]
+
+include::modules/customize-certificates-understanding-service-serving.adoc[leveloffset=+1]
+
+[IMPORTANT]
```

Comment:
I would love for it to be there, but this is a dirty hack because we can't include xrefs inside of modules. I think a link to this section is required, and the only way to do it is to include it in the assembly instead of the module. :(

Reply:
That makes me a sad panda! Thanks for the explanation :slightly_smiling_face: